spec.html

```diff
@@ -10,7 +10,7 @@
   }
 </script>
 <pre class=metadata>
-  title: ECMAScript® 2016 Language Specification
```

Comment:
I'm asking about your change. I think using literal unicode characters in the ecmarkup is great, and the build process should be responsible for changing it to an HTML entity.